provisioning/inventory/hosts

```diff
@@ -1,6 +1,6 @@
 [devservers]
 # TODO: relative path to private key breaks ability to run vagrant up from anywhere in the project
-vagrant ansible_ssh_host=127.0.0.1 ansible_ssh_port=2222 ansible_ssh_user=vagrant ansible_ssh_private_key_file=./.vagrant/machines/default/virtualbox/private_key
+vagrant ansible_ssh_host=127.0.0.1 ansible_ssh_port=2222 ansible_ssh_user=vagrant
```

Comment:
I don't know how. maybe it uses my generic key that's in the default location?
it works great. every time I run the "vagrant provision" command, things work like it should, and I get a play recap and everything eventually. 
when i add this line - it just breaks down.